core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/Utils.java

```diff
@@ -91,6 +95,30 @@ public static ShaclSail getInitializedShaclSail(String shapeData) throws IOExcep
 		return sail;
 	}
 
+	public static ShaclSail getInitializedShaclSailNativeStore(TemporaryFolder file, String shapeData)
+			throws IOException {
+		NativeStore baseSail = getNativeStore(file);
+
+		ShaclSail sail = new ShaclSail(baseSail);
+		Utils.loadShapeData(sail, shapeData);
+		return sail;
+	}
+
+	public static Sail getTestNotifyingSailNativeStore(TemporaryFolder file) {
+		NativeStore baseSail = getNativeStore(file);
+
+		return new TestNotifyingSail(baseSail);
+	}
+
+	private static NativeStore getNativeStore(TemporaryFolder file) {
+		NativeStore baseSail = new NativeStore(file.getFile());
+		baseSail.setValueCacheSize(100);
+		baseSail.setNamespaceCacheSize(100);
+		baseSail.setValueIDCacheSize(100);
+		baseSail.setNamespaceIDCacheSize(100);
```

Comment:
If you want realistic usage benmarking, should you be tweaking these settings? I doubt many of our users do.